pfcwd/main.py

```diff
@@ -194,9 +194,28 @@ def interval(poll_interval):
     configdb.connect()
     pfcwd_info = {}
     if poll_interval is not None:
+        pfcwd_table = configdb.get_table(CONFIG_DB_PFC_WD_TABLE_NAME)
+        entry_min = 3000
+        for entry in pfcwd_table:
+            if("Ethernet" not in entry):
+                continue
+            detection_time_entry_value = int(configdb.get_entry(CONFIG_DB_PFC_WD_TABLE_NAME, entry).get('detection_time'))
+            restoration_time_entry_value = int(configdb.get_entry(CONFIG_DB_PFC_WD_TABLE_NAME, entry).get('restoration_time'))
+            if ((detection_time_entry_value != None) and (detection_time_entry_value < entry_min)):
+                entry_min = detection_time_entry_value
+                res_str = "detection_time"
+            if ((restoration_time_entry_value != None) and (restoration_time_entry_value < entry_min)):
+                entry_min = restoration_time_entry_value
+                res_str = "restoration_time"
+        if entry_min < poll_interval:
+            poll_interval = entry_min - 1
+            print "polling_interval is greater than {}, using polling_interval = {}ms".format(res_str,poll_interval)
```

Comment:
output should go to stderr, and also return error code 1

Reply:
@lguohan 
at this point we are using a different **valid** value for the polling interval.
you think we should not allow it? if the input is not valid just exit without using an alternative value?

Reply:
It's better to exit at this point and allow the user to decide the value

Reply:
@lguohan 
please review